app/controllers/posts_controller.rb

```diff
@@ -7,4 +7,24 @@ def index
   def show
     @post = Post.find(params[:id])
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
```

Comment:
- [ ] Great working on your Post `create method`, kindly check and fix the issue with the redirection after the post is created it throws the error in the screenshot, you can change the redirect to this `redirect_to user_path(current_user)`.

![edrror](https://user-images.githubusercontent.com/48371156/185739714-f1105883-7360-4e44-97ad-e9998eb116c3.png)